presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -366,6 +367,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         "Experimental: enable complex filter pushdown",
                         hiveClientConfig.isPushdownFilterEnabled(),
                         false),
+                booleanProperty(
```

Comment:
Usually I see new properties were added to the end of the list, not in the middle, so people can see the order they're added. I'm not sure if this is the case? cc @wenleix

Reply:
I wanted to keep this property next to the other related property

Reply:
I see both cases happened (added to the end of list vs. added next to related properties). I don't have strong opinions on this.